test/integration/worker_inference_and_forecast_test.go

```diff
@@ -44,38 +44,33 @@ func getNonZeroTopicEpochLastRan(m testCommon.TestConfig, topicID uint64, maxRet
 	return nil, errors.New("topicEpochLastRan is still 0 after retrying")
 }
 
-func InsertSingleWorkerBulk(m testCommon.TestConfig, topic *types.Topic, blockHeight int64) {
+func InsertSingleWorkerPayload(m testCommon.TestConfig, topic *types.Topic, blockHeight int64) {
```

Comment:
just a note, when you merge back from ora-2015, you'll find many changes on integration folder - please keep those.